gcloud_bigtable/row.py

```diff
@@ -416,12 +419,13 @@ def _commit_mutate(self, timeout_seconds=None):
             mutations=mutations_list,
         )
         timeout_seconds = timeout_seconds or self.timeout_seconds
-        response = self.client.data_stub.MutateRow.async(request_pb,
-                                                         timeout_seconds)
         # We expect a `._generated.empty_pb2.Empty`.
-        response.result()
+        if async:
+            self.client.data_stub.MutateRow.async(request_pb, timeout_seconds).result()
```

Comment:
The commit method would randomly (after a couple of (ten) thousand writes) never return (so not even after the timeout).  It happened fairly consistent; every couple of hours or so. After we switched to the sync version (and that was the only change we made), we didn't run into the problem anymore. We're using GRPC 0.11.